lib/commands/validate-spec.ts

```diff
@@ -21,9 +22,11 @@ export async function handler(argv: yargs.Arguments): Promise<void> {
       pretty: argv.p
     }
     if (specPath.match(/.*composite.*/gi) !== null) {
-      await validate.validateCompositeSpec(specPath, vOptions)
+      const result = await validate.validateCompositeSpec(specPath, vOptions)
+      return iterator.some(result, v => !v.validityStatus) ? 1 : 0
```

Comment:
could similar logic as my previous comment be applied here? is it required to check on validity status for each result?

Reply:
No, if at least one is failed the job has to fail.